deployments/get/objects.go

```diff
@@ -368,6 +368,7 @@ func GenerateDaemonSet(env, namespace string) *appsv1.DaemonSet {
 										"IPC_LOCK",
 										"CAP_DAC_OVERRIDE",
 										"CAP_DAC_READ_SEARCH",
+										"CAP_AUDIT_READ",
```

Comment:
Why we need this ?

Reply:
it's not required anymore, removed, thanks for pointing this out.